icerate.py

```diff
@@ -435,7 +435,7 @@ def print_syntax():
     optdict = dict(optlist)
 
     try:
-        infile = optdict.get("-f", args[0])
+        infile = optdict.get("-f", sys.argv[1])
```

Comment:
In this case, the original behaviour was to use the first argument not a part of a command line switch as the filename, rather than the first argument period. A hypothetical example of where this would matter if if you tried to do

    icerate.py -L 1 data.h5